packages/webapp/.eslintrc.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": [
-    "next/core-web-vitals",
-    "plugin:storybook/recommended"
```

Comment:
This legacy version of Storybook was really getting in the way progress, especially with TS, so I just ripped it al out for now. Version 7 and 8 are completely new re-writes so maybe we can check them out in the future.